cfgov/unprocessed/css/forms.less

```diff
@@ -607,44 +398,346 @@
         .grid_column(4);
     });
 }
 
 .form-l_col-2-3 {
-    .respond-to-min(1060px, {
+    .respond-to-min(@bp-med-min, {
         .grid_column(8);
     });
 }
 
+.form-l_col-1-4 {
+    .respond-to-min(@bp-med-min, {
+        .grid_column(3);
+    });
+}
+
 .form-l_col-1 {
-    .respond-to-min(1060px, {
+    .respond-to-min(@bp-med-min, {
         .grid_column(12);
     });
 }
 
+
+/* topdoc
+  name: Checkboxes and radio buttons
+    family: cf-forms
+    notes:
+      - "CSS for checkboxes and radio buttons requires following a specific HTML structure for each type of input, otherwise the styles will not be properly applied."
+      - "You MUST wrap your input.cf-input in a label."
+      - "Remember to use 'for' and 'id' attributes so that selected/checked styles function properly."
+      - "Remember to use the same 'name' attribute for each set of radio buttons."
+    patterns:
+      - name: Checkbox
+        markup: |
+          <label class="cf-input_label">
+              <input class="cf-input cf-input_checkbox" type="checkbox" name="option1" value="Option 1">
+              Option 1
+          </label>
+          <br>
+          <label class="cf-input_label">
+              <input class="cf-input cf-input_checkbox" type="checkbox" name="option2" value="Option 2">
+              Option 2
+          </label>
+        codenotes:
+          - |
+            Structural cheat sheet:
+            -----------------------
+            label.cf-input_label
+              input[type=checkbox].cf-input
+        notes:
+          - "Use the markup structure above to add checkbox .cf-input elements."
+      - name: Radio buttons
+        markup: |
+          <label class="cf-input_label" for"yes">
+              <span class="cf-input_text">Yes</span>
+              <input class="cf-input cf-input_hidden" type="radio" name="cf-input-example_default" id="yes">
+              <span class="cf-input_radio"></span>
+          </label>
+          <br>
+          <label class="cf-input_label" for="no">
+              <span class="cf-input_text">No</span>
+              <input class="cf-input cf-input_hidden" type="radio" name="cf-input-example_default" id="no">
+              <span class="cf-input_radio"></span>
+          </label>
+        codenotes:
+          - |
+            Structural cheat sheet:
+            -----------------------
+            label.cf-input_label
+              span.cf-input_text
+              input[type=radio].cf-input.cf-input_hidden
+              span.cf-input_radio
+        notes:
+          - "Use the markup structure above to add radio button .cf-input elements."
+          - "Radio buttons require extra markup to apply custom styles."
+    tags:
+      - cf-forms
+*/
+
+@input-active:                  @pacific;
+
+@input-bg-disabled:             @gray;
+@input-border-disabled:         @gray-50;
+
+// Insets
+@input-inset-bg:                @gray-20;
+@input-inset-selected:          @pacific-20;
+
+
+input.cf-input {
```

Comment:
I thought it was necessary for the `input[type=` selectors, but it isn't! Yay, lower-level selectors.